src/reducers/books.js

```diff
@@ -0,0 +1,16 @@
+const CREATE_BOOK = 'CREATE_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
**OPTIONAL**: You can export these constants and use them on your actions creators to avoid typos.

Reply:
Thank you for your recommendation!